src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -321,6 +321,9 @@ public static class ClientOptions {
                 "13819135 (0xD2DCFF in decimal) is the classic blue from GT5 (default)."})
         public int defaultUIColor = 0xD2DCFF;
 
+        @Config.Comment("Whether some tooltips should be blinking for better visibility")
+        public boolean blinkingTooltips = true;
```

Comment:
I think this should be written and work other way around. Right now it sounds like we have some blinking BS which we want to puh on everyone. If you write it like "Prevent tooltips from blinking and rapidly switching colors." Then it will sound more like accessibility option which technically it is.